OpenKh.Tools.Kh2TextEditor/ViewModels/TextEditorViewModel.cs

```diff
@@ -127,6 +139,66 @@ public TextEditorViewModel()
             Drawing = new SpriteDrawingDirect3D();
             CurrentMessageEncoder = Encoders.InternationalSystem;
             _messages = new MessagesModel(this, this, new Msg.Entry[] { });
+
+            HandleReturn = new RelayCommand(x =>
+            {
+                Text += "{:newline}";                
+                SelectedItem = SelectedItem;
+            });
+
+            HandleRemoval = new RelayCommand(x =>
+            {
+                MessageEntries.RemoveAll(x => x.Id == SelectedItem.Id);
+                ResetMessagesView();
```

Comment:
I do not ask any action here, but this makes me realise how unflexible this tool is. `ResetMessagesView` resets the list and de-select the currently selected item.

What we do in other tools is to use a single list for everything. But this tool uses two separate lists: the effective list of messages and the list shown in the ListBox. 🤮